File: OpenBots.Server.Model/File/StorageDrive.cs
using OpenBots.Server.Model.Core;
using System;

namespace OpenBots.Server.Model.File
{
    public class StorageDrive : NamedEntity
    {
        public string FileStorageAdapterType { get; set; }
        public Guid? OrganizationId { get; set; }
        public long? StorageSizeInBytes { get; set; }
        public string StoragePath { get; set; }
    }
}

File: OpenBots.Server.Web/ClientApp/src/app/interfaces/storageDrive.ts
export interface StorageDrive {
  id: string;
  isDeleted: false;
  createdBy?: string;
  createdOn?: string;
  deletedBy?: string;
  deleteOn?: string;
  timestamp?: string;
  updatedOn?: string;
  updatedBy?: string;
  name: string;
  fileStorageAdapterType: string;
  organizationId: string;
  storagePath: string;
  storageSizeInBytes: number;
  maxStorageAllowedInBytes: number;
  isDefault: boolean;
}
